packages/language-support/src/antlr-grammar/CypherParser.g4

```diff
@@ -1553,16 +1602,22 @@ symbolicNameOrStringParameter
    | parameter["STRING"]
    ;
 
+aliasName:
+    symbolicAliasNameOrParameter;
+
+databaseName:
+    symbolicAliasNameOrParameter;
+
```

Comment:
These are the real changes to the grammar, the rest are just bringing it up to date